.github/workflows/nightly.yml

```diff
@@ -192,6 +193,7 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.CORE_PREVIEWS_S3_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.CORE_PREVIEWS_S3_SECRET_KEY }}
         run: |
+          sudo apt update
```

Comment:
nit: I think the apt update above is probably enough... but it might also not hurt if we have it two times